desktop/src/main/resources/bisq.css

```diff
@@ -233,6 +249,24 @@
     -fx-font-size: 1.35em;
 }
 
+.bisq-text-2 {
+    -fx-text-fill: -bisq-grey-13;
+    -fx-font-family: "IBM Plex Sans Regular";
```

Comment:
I guess we dont need to define the -fx-font-family as its the default

Reply:
Good point. I see two "Regular" font faces defined on the top of `bisq.css` - [Sans](https://github.com/bisq-network/bisq2/blob/24e8e0a8b75060b168abc660017303668e4b8dbd/desktop/src/main/resources/bisq.css#L1-L3) and [Mono](https://github.com/bisq-network/bisq2/blob/24e8e0a8b75060b168abc660017303668e4b8dbd/desktop/src/main/resources/bisq.css#L17-L19) - I'll check how javafx chooses one of them. I wonder if we don't need to define one of theme somewhere in the root level.

Reply:
Seems like you're right - [fixed](https://github.com/bisq-network/bisq2/pull/296/commits/402cf2f1436736ce55cddb6c448be217f65b5963). 

Unfortunately I have no idea how JavaFX load default font - hope I'll figure it out one day :)